### ISSUE TITLE
[DOC]: Usage of the training/tuning scripts

### ISSUE THREAD

**Issue by MrH100000** [nv-morpheus/Morpheus#892]
### How would you describe the priority of this documentation request

Medium

### Describe the future/missing documentation

I'm attempting to train/tune the phishing model with custom data. I'm able to successfully run the script, which produces the pytorch_model.bin and config.json files for the model. However I'm quite unsure how to actually take this and create the onnx format model to use with Triton. I also looked through the training/tuning scripts for other models, and I couldn't figure out how to generate the onnx model for any of them.

I wish suggest that it might be helpful to have some documentation for how to run the training scripts, and most importantly how to use the result. I ask this mainly because I am failing to understand how to use the training/tuning scripts, but it might help others too.

### Where have you looked?

https://github.com/nv-morpheus/Morpheus/blob/branch-23.07/models/README.md
https://github.com/nv-morpheus/Morpheus/tree/branch-23.07/models/training-tuning-scripts/phishing-models
https://github.com/nv-morpheus/Morpheus/tree/branch-23.07/models/triton-model-repo
https://github.com/nv-morpheus/Morpheus/tree/branch-23.07/examples/developer_guide/2_1_real_world_phishing

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

**Comment by MrH100000** [comment]
The documentation added to the notebook in #903 is definitely helpful! Although I'm not using the notebook, I was able to use the information to create the ONNX version for Triton in the script. Thank you!
